holoviews/plotting/bokeh/chart.py

```diff
@@ -72,11 +72,13 @@ class PointPlot(LegendPlot, ColorbarPlot):
     _batched_style_opts = line_properties + fill_properties + ['size', 'marker', 'angle']
 
     def _init_glyph(self, plot, mapping, properties):
-        if "radius" in properties:
+        if properties.get("radius") is not None:
```

Comment:
```suggestion
        if properties.get("radius"):
```

Will also disable it for `0` or `False`, not sure about this.

Reply:
I'd leave it as is.